WordPress/src/main/java/org/wordpress/android/ui/sitecreation/verticals/VerticalsScreenListener.kt

```diff
@@ -0,0 +1,5 @@
+package org.wordpress.android.ui.sitecreation.verticals
+
+interface VerticalsScreenListener {
```

Comment:
I think it'd be worth having a separate `onSkip` method here and making the `verticalId` one not nullable. Ideally these methods should do only one thing.

Reply:
Yes, I had this feeling too. I created a new interface so we don't have bunch of `onSkip()` methods.